lib/rules/indent.js

```diff
@@ -1039,6 +1039,11 @@ module.exports = {
 
             VariableDeclarator(node) {
                 if (node.init) {
+                    const equalOperator = sourceCode.getTokenBefore(node.init, astUtils.isNotOpeningParenToken);
+
+                    if (equalOperator) {
```

Comment:
I thought that since getTokenBefore() can return null, and ignoreToken() cannot handle null values, I should have a check. In this case it should always return the equal operator.

I will remove the if statement to clean this up.